server/bin/unittests

```diff
@@ -435,9 +435,8 @@ function _run_test {
     _save_tree
     _dump_logs
     _verify_output ${testname}
-    res=$?
     _reset_state ${testname}
-    return $res
+    return 0
```

Comment:
It is no longer necessary.  If we look at [`_verify_output`](https://github.com/distributed-system-analysis/pbench/blob/496ab147894858ab165f1e0c5823b8a940bbd843/server/bin/unittests#L216) we see that where it would return `1` or `0` is where is also writes `FAIL` or `PASS` to the `$_testres` file.  So returning that status and checking for it is not necessary.  Instead, at line [738](https://github.com/distributed-system-analysis/pbench/blob/496ab147894858ab165f1e0c5823b8a940bbd843/server/bin/unittests#L738), we just have to check for `PASS` in `$_testres` to know if the test passed or not.